### ISSUE TITLE
Why does Twitter set message url?

### ISSUE THREAD

**Issue by rpk7524** [ithunt/asksg#10]
See https://github.com/ithunt/asksg/blob/master/src/main/java/edu/rit/asksg/domain/Twitter.java#L56

Would this be more appropriate to be set on the Conversation (or is this url going to be already encapsulated within U Service,Provider.Identity and thus unneeded entirely)?  Twitter is the only caller for url in message.

**Comment by ithunt** [comment]
You're right, currently Twitter is the only one that uses message.url. For facebook or reddit, sending the user to the URL of the conversation is appropriate. However, twitter does not have conversations and thus messages have individual urls, 'conversations' may be spread out among several twitter home pages

**Comment by rpk7524** [comment]
Twitter has Conversations created by context. Calls to expanded/batch/id/ with facepile_max:[int], include[]:social_proof, include[]:ancestors, include[]:descendants provide a close enough parallel to our conversation concept.  Also, doesn't show/ against the tweet id provide the same functionality as the url?
